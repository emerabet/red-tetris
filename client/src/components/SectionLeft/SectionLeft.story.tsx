import React from 'react';
import { storiesOf } from '@storybook/react';
import SectionLeft from './SectionLeft';

storiesOf('SectionLeft', module)
   .add('room player', () => (
      <SectionLeft
         room="SUPERROOM"
         player="BIG BOSS"
      />
   ));
