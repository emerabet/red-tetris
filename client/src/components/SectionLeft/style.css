.sectionLeft {
    max-width: 25%;
    min-width: 250px;
   
    display: flex;
    flex-grow: 1;
    flex-direction: column;
    justify-content: flex-start;
    align-items: center;
}