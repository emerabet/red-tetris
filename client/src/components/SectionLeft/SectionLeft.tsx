import React from 'react';
import RedTetris from '../RedTetris';
import LabeledBox from '../LabeledBox';
import NextPieces from '../NextPieces';
import AdminButton from '../AdminButton';
import Score from '../Score';
import * as utils from '../../Utils/constants';

import './style.css';

interface SectionLeftProps {
  room: string;
  player: string;
  status: string;
  score: number;
}

const SectionLeft: React.SFC<SectionLeftProps> = (props) => {

  return (
    <div className="sectionLeft">
      <RedTetris additionalClassName="small" />
      <Score
        score={props.score}
      />
      <LabeledBox
        label="room:"
        content={props.room}
      />
      <LabeledBox
        label="player:"
        content={props.player}
      />
      <AdminButton
        text={props.status}
      />
    </div>
  );
};

export default SectionLeft;
