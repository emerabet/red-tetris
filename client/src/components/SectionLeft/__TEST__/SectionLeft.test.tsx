import React from 'react';

import renderer from 'react-test-renderer';
import SectionLeft from '../SectionLeft';

it('renders correctly with defaults', () => {
  const sectionLeft = renderer.create(<SectionLeft
    room="SUPERROOM"
    player="BIG BOSS"
    status="start"
    level={2}
    score={8}
  />).toJSON();
  expect(sectionLeft).toMatchSnapshot();
});
