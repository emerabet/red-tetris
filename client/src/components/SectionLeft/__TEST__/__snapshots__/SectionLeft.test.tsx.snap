// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with defaults 1`] = `
<div
  className="sectionLeft"
>
  <div
    className="sectionLeftInside"
  >
    <div
      className="redTetris small"
    >
      <div
        className="box boxR"
      >
        <div
          className="boxCube boxR1"
        />
        <div
          className="boxCube boxR2"
        />
        <div
          className="boxCube boxR3"
        />
        <div
          className="boxCube boxR4"
        />
        <div
          className="boxCube boxR5"
        />
        <div
          className="boxCube boxR6"
        />
        <div
          className="boxCube boxR7"
        />
        <div
          className="boxCube boxR8"
        />
        <div
          className="boxCube boxR9"
        />
        <div
          className="boxCube boxR10"
        />
        <div
          className="boxCube boxR11"
        />
        <div
          className="boxCube boxR12"
        />
        <div
          className="boxCube boxR13"
        />
        <div
          className="boxCube boxR14"
        />
      </div>
      <div
        className="box boxE"
      >
        <div
          className="boxCube boxE1"
        />
        <div
          className="boxCube boxE2"
        />
        <div
          className="boxCube boxE3"
        />
        <div
          className="boxCube boxE4"
        />
        <div
          className="boxCube boxE5"
        />
        <div
          className="boxCube boxE6"
        />
        <div
          className="boxCube boxE7"
        />
        <div
          className="boxCube boxE8"
        />
        <div
          className="boxCube boxE9"
        />
        <div
          className="boxCube boxE10"
        />
        <div
          className="boxCube boxE11"
        />
        <div
          className="boxCube boxE12"
        />
        <div
          className="boxCube boxE13"
        />
      </div>
      <div
        className="box boxD"
      >
        <div
          className="boxCube boxD1"
        />
        <div
          className="boxCube boxD2"
        />
        <div
          className="boxCube boxD3"
        />
        <div
          className="boxCube boxD4"
        />
        <div
          className="boxCube boxD5"
        />
        <div
          className="boxCube boxD6"
        />
        <div
          className="boxCube boxD7"
        />
        <div
          className="boxCube boxD8"
        />
        <div
          className="boxCube boxD9"
        />
        <div
          className="boxCube boxD10"
        />
        <div
          className="boxCube boxD11"
        />
        <div
          className="boxCube boxD12"
        />
      </div>
      <div
        className="titleSpace"
      />
      <div
        className="boxs boxT"
      >
        <div
          className="boxCube boxT1"
        />
        <div
          className="boxCube boxT2"
        />
        <div
          className="boxCube boxT3"
        />
        <div
          className="boxCube boxT4"
        />
        <div
          className="boxCube boxT5"
        />
        <div
          className="boxCube boxT6"
        />
        <div
          className="boxCube boxT7"
        />
      </div>
      <div
        className="box boxE"
      >
        <div
          className="boxCube boxE1"
        />
        <div
          className="boxCube boxE2"
        />
        <div
          className="boxCube boxE3"
        />
        <div
          className="boxCube boxE4"
        />
        <div
          className="boxCube boxE5"
        />
        <div
          className="boxCube boxE6"
        />
        <div
          className="boxCube boxE7"
        />
        <div
          className="boxCube boxE8"
        />
        <div
          className="boxCube boxE9"
        />
        <div
          className="boxCube boxE10"
        />
        <div
          className="boxCube boxE11"
        />
        <div
          className="boxCube boxE12"
        />
        <div
          className="boxCube boxE13"
        />
      </div>
      <div
        className="boxs boxT"
      >
        <div
          className="boxCube boxT1"
        />
        <div
          className="boxCube boxT2"
        />
        <div
          className="boxCube boxT3"
        />
        <div
          className="boxCube boxT4"
        />
        <div
          className="boxCube boxT5"
        />
        <div
          className="boxCube boxT6"
        />
        <div
          className="boxCube boxT7"
        />
      </div>
      <div
        className="box boxR"
      >
        <div
          className="boxCube boxR1"
        />
        <div
          className="boxCube boxR2"
        />
        <div
          className="boxCube boxR3"
        />
        <div
          className="boxCube boxR4"
        />
        <div
          className="boxCube boxR5"
        />
        <div
          className="boxCube boxR6"
        />
        <div
          className="boxCube boxR7"
        />
        <div
          className="boxCube boxR8"
        />
        <div
          className="boxCube boxR9"
        />
        <div
          className="boxCube boxR10"
        />
        <div
          className="boxCube boxR11"
        />
        <div
          className="boxCube boxR12"
        />
        <div
          className="boxCube boxR13"
        />
        <div
          className="boxCube boxR14"
        />
      </div>
      <div
        className="boxs boxI"
      >
        <div
          className="boxCube boxI1"
        />
        <div
          className="boxCube boxI2"
        />
        <div
          className="boxCube boxI3"
        />
        <div
          className="boxCube boxI4"
        />
        <div
          className="boxCube boxI5"
        />
        <div
          className="boxCube boxI6"
        />
        <div
          className="boxCube boxI7"
        />
        <div
          className="boxCube boxI8"
        />
        <div
          className="boxCube boxI9"
        />
      </div>
      <div
        className="box boxS"
      >
        <div
          className="boxCube boxS1"
        />
        <div
          className="boxCube boxS2"
        />
        <div
          className="boxCube boxS3"
        />
        <div
          className="boxCube boxS4"
        />
        <div
          className="boxCube boxS5"
        />
        <div
          className="boxCube boxS6"
        />
        <div
          className="boxCube boxS7"
        />
        <div
          className="boxCube boxS8"
        />
        <div
          className="boxCube boxS9"
        />
        <div
          className="boxCube boxS10"
        />
        <div
          className="boxCube boxS11"
        />
        <div
          className="boxCube boxS12"
        />
        <div
          className="boxCube boxS13"
        />
        <div
          className="boxCube boxS14"
        />
      </div>
    </div>
    <div
      className="scoreContainer"
    >
      <div>
        LEVEL: 
        2
      </div>
      <div>
        SCORE: 
        8
      </div>
    </div>
    <div
      className="outerLabel"
    >
      <div
        className="labelTitle "
      >
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube r1"
            />
            <div
              className="labelCube r2"
            />
            <div
              className="labelCube r3"
            />
            <div
              className="labelCube r4"
            />
            <div
              className="labelCube r5"
            />
            <div
              className="labelCube r6"
            />
            <div
              className="labelCube r7"
            />
            <div
              className="labelCube r8"
            />
            <div
              className="labelCube r9"
            />
            <div
              className="labelCube r10"
            />
            <div
              className="labelCube r11"
            />
            <div
              className="labelCube r12"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube o1"
            />
            <div
              className="labelCube o2"
            />
            <div
              className="labelCube o3"
            />
            <div
              className="labelCube o4"
            />
            <div
              className="labelCube o5"
            />
            <div
              className="labelCube o6"
            />
            <div
              className="labelCube o7"
            />
            <div
              className="labelCube o8"
            />
            <div
              className="labelCube o9"
            />
            <div
              className="labelCube o10"
            />
            <div
              className="labelCube o11"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube o1"
            />
            <div
              className="labelCube o2"
            />
            <div
              className="labelCube o3"
            />
            <div
              className="labelCube o4"
            />
            <div
              className="labelCube o5"
            />
            <div
              className="labelCube o6"
            />
            <div
              className="labelCube o7"
            />
            <div
              className="labelCube o8"
            />
            <div
              className="labelCube o9"
            />
            <div
              className="labelCube o10"
            />
            <div
              className="labelCube o11"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetter"
          >
            <div
              className="labelCube m1"
            />
            <div
              className="labelCube m2"
            />
            <div
              className="labelCube m3"
            />
            <div
              className="labelCube m4"
            />
            <div
              className="labelCube m5"
            />
            <div
              className="labelCube m6"
            />
            <div
              className="labelCube m7"
            />
            <div
              className="labelCube m8"
            />
            <div
              className="labelCube m9"
            />
            <div
              className="labelCube m10"
            />
            <div
              className="labelCube m11"
            />
            <div
              className="labelCube m12"
            />
            <div
              className="labelCube m13"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterS"
          >
            <div
              className="labelCube dot-1"
            />
            <div
              className="labelCube dot-2"
            />
          </div>
           
        </div>
      </div>
      <div
        className="labelTitle paddingLabel"
      >
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube s1"
            />
            <div
              className="labelCube redCube s2"
            />
            <div
              className="labelCube redCube s3"
            />
            <div
              className="labelCube redCube s4"
            />
            <div
              className="labelCube redCube s5"
            />
            <div
              className="labelCube redCube s6"
            />
            <div
              className="labelCube redCube s7"
            />
            <div
              className="labelCube redCube s8"
            />
            <div
              className="labelCube redCube s9"
            />
            <div
              className="labelCube redCube s10"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube u1"
            />
            <div
              className="labelCube redCube u2"
            />
            <div
              className="labelCube redCube u3"
            />
            <div
              className="labelCube redCube u4"
            />
            <div
              className="labelCube redCube u5"
            />
            <div
              className="labelCube redCube u6"
            />
            <div
              className="labelCube redCube u7"
            />
            <div
              className="labelCube redCube u8"
            />
            <div
              className="labelCube redCube u9"
            />
            <div
              className="labelCube redCube u10"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube p1"
            />
            <div
              className="labelCube redCube p2"
            />
            <div
              className="labelCube redCube p3"
            />
            <div
              className="labelCube redCube p4"
            />
            <div
              className="labelCube redCube p5"
            />
            <div
              className="labelCube redCube p6"
            />
            <div
              className="labelCube redCube p7"
            />
            <div
              className="labelCube redCube p8"
            />
            <div
              className="labelCube redCube p9"
            />
            <div
              className="labelCube redCube p10"
            />
            <div
              className="labelCube redCube p11"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube e1"
            />
            <div
              className="labelCube redCube e2"
            />
            <div
              className="labelCube redCube e3"
            />
            <div
              className="labelCube redCube e4"
            />
            <div
              className="labelCube redCube e5"
            />
            <div
              className="labelCube redCube e6"
            />
            <div
              className="labelCube redCube e7"
            />
            <div
              className="labelCube redCube e8"
            />
            <div
              className="labelCube redCube e9"
            />
            <div
              className="labelCube redCube e10"
            />
            <div
              className="labelCube redCube e11"
            />
            <div
              className="labelCube redCube e12"
            />
            <div
              className="labelCube redCube e13"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube r1"
            />
            <div
              className="labelCube redCube r2"
            />
            <div
              className="labelCube redCube r3"
            />
            <div
              className="labelCube redCube r4"
            />
            <div
              className="labelCube redCube r5"
            />
            <div
              className="labelCube redCube r6"
            />
            <div
              className="labelCube redCube r7"
            />
            <div
              className="labelCube redCube r8"
            />
            <div
              className="labelCube redCube r9"
            />
            <div
              className="labelCube redCube r10"
            />
            <div
              className="labelCube redCube r11"
            />
            <div
              className="labelCube redCube r12"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube r1"
            />
            <div
              className="labelCube redCube r2"
            />
            <div
              className="labelCube redCube r3"
            />
            <div
              className="labelCube redCube r4"
            />
            <div
              className="labelCube redCube r5"
            />
            <div
              className="labelCube redCube r6"
            />
            <div
              className="labelCube redCube r7"
            />
            <div
              className="labelCube redCube r8"
            />
            <div
              className="labelCube redCube r9"
            />
            <div
              className="labelCube redCube r10"
            />
            <div
              className="labelCube redCube r11"
            />
            <div
              className="labelCube redCube r12"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube o1"
            />
            <div
              className="labelCube redCube o2"
            />
            <div
              className="labelCube redCube o3"
            />
            <div
              className="labelCube redCube o4"
            />
            <div
              className="labelCube redCube o5"
            />
            <div
              className="labelCube redCube o6"
            />
            <div
              className="labelCube redCube o7"
            />
            <div
              className="labelCube redCube o8"
            />
            <div
              className="labelCube redCube o9"
            />
            <div
              className="labelCube redCube o10"
            />
            <div
              className="labelCube redCube o11"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube o1"
            />
            <div
              className="labelCube redCube o2"
            />
            <div
              className="labelCube redCube o3"
            />
            <div
              className="labelCube redCube o4"
            />
            <div
              className="labelCube redCube o5"
            />
            <div
              className="labelCube redCube o6"
            />
            <div
              className="labelCube redCube o7"
            />
            <div
              className="labelCube redCube o8"
            />
            <div
              className="labelCube redCube o9"
            />
            <div
              className="labelCube redCube o10"
            />
            <div
              className="labelCube redCube o11"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetter"
          >
            <div
              className="labelCube redCube m1"
            />
            <div
              className="labelCube redCube m2"
            />
            <div
              className="labelCube redCube m3"
            />
            <div
              className="labelCube redCube m4"
            />
            <div
              className="labelCube redCube m5"
            />
            <div
              className="labelCube redCube m6"
            />
            <div
              className="labelCube redCube m7"
            />
            <div
              className="labelCube redCube m8"
            />
            <div
              className="labelCube redCube m9"
            />
            <div
              className="labelCube redCube m10"
            />
            <div
              className="labelCube redCube m11"
            />
            <div
              className="labelCube redCube m12"
            />
            <div
              className="labelCube redCube m13"
            />
          </div>
           
        </div>
      </div>
    </div>
    <div
      className="outerLabel"
    >
      <div
        className="labelTitle "
      >
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube p1"
            />
            <div
              className="labelCube p2"
            />
            <div
              className="labelCube p3"
            />
            <div
              className="labelCube p4"
            />
            <div
              className="labelCube p5"
            />
            <div
              className="labelCube p6"
            />
            <div
              className="labelCube p7"
            />
            <div
              className="labelCube p8"
            />
            <div
              className="labelCube p9"
            />
            <div
              className="labelCube p10"
            />
            <div
              className="labelCube p11"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube l1"
            />
            <div
              className="labelCube l2"
            />
            <div
              className="labelCube l3"
            />
            <div
              className="labelCube l4"
            />
            <div
              className="labelCube l5"
            />
            <div
              className="labelCube l6"
            />
            <div
              className="labelCube l7"
            />
            <div
              className="labelCube l8"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube a1"
            />
            <div
              className="labelCube a2"
            />
            <div
              className="labelCube a3"
            />
            <div
              className="labelCube a4"
            />
            <div
              className="labelCube a5"
            />
            <div
              className="labelCube a6"
            />
            <div
              className="labelCube a7"
            />
            <div
              className="labelCube a8"
            />
            <div
              className="labelCube a9"
            />
            <div
              className="labelCube a10"
            />
            <div
              className="labelCube a11"
            />
            <div
              className="labelCube a12"
            />
            <div
              className="labelCube a13"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube y1"
            />
            <div
              className="labelCube y2"
            />
            <div
              className="labelCube y3"
            />
            <div
              className="labelCube y4"
            />
            <div
              className="labelCube y5"
            />
            <div
              className="labelCube y6"
            />
            <div
              className="labelCube y7"
            />
            <div
              className="labelCube y8"
            />
            <div
              className="labelCube y9"
            />
            <div
              className="labelCube y10"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube e1"
            />
            <div
              className="labelCube e2"
            />
            <div
              className="labelCube e3"
            />
            <div
              className="labelCube e4"
            />
            <div
              className="labelCube e5"
            />
            <div
              className="labelCube e6"
            />
            <div
              className="labelCube e7"
            />
            <div
              className="labelCube e8"
            />
            <div
              className="labelCube e9"
            />
            <div
              className="labelCube e10"
            />
            <div
              className="labelCube e11"
            />
            <div
              className="labelCube e12"
            />
            <div
              className="labelCube e13"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube r1"
            />
            <div
              className="labelCube r2"
            />
            <div
              className="labelCube r3"
            />
            <div
              className="labelCube r4"
            />
            <div
              className="labelCube r5"
            />
            <div
              className="labelCube r6"
            />
            <div
              className="labelCube r7"
            />
            <div
              className="labelCube r8"
            />
            <div
              className="labelCube r9"
            />
            <div
              className="labelCube r10"
            />
            <div
              className="labelCube r11"
            />
            <div
              className="labelCube r12"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterS"
          >
            <div
              className="labelCube dot-1"
            />
            <div
              className="labelCube dot-2"
            />
          </div>
           
        </div>
      </div>
      <div
        className="labelTitle paddingLabel"
      >
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube b1"
            />
            <div
              className="labelCube redCube b2"
            />
            <div
              className="labelCube redCube b3"
            />
            <div
              className="labelCube redCube b4"
            />
            <div
              className="labelCube redCube b5"
            />
            <div
              className="labelCube redCube b6"
            />
            <div
              className="labelCube redCube b7"
            />
            <div
              className="labelCube redCube b8"
            />
            <div
              className="labelCube redCube b9"
            />
            <div
              className="labelCube redCube b10"
            />
            <div
              className="labelCube redCube b11"
            />
            <div
              className="labelCube redCube b12"
            />
            <div
              className="labelCube redCube b13"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterS"
          >
            <div
              className="labelCube redCube i1"
            />
            <div
              className="labelCube redCube i2"
            />
            <div
              className="labelCube redCube i3"
            />
            <div
              className="labelCube redCube i4"
            />
            <div
              className="labelCube redCube i5"
            />
            <div
              className="labelCube redCube i6"
            />
            <div
              className="labelCube redCube i7"
            />
            <div
              className="labelCube redCube i8"
            />
            <div
              className="labelCube redCube i9"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube g1"
            />
            <div
              className="labelCube redCube g2"
            />
            <div
              className="labelCube redCube g3"
            />
            <div
              className="labelCube redCube g4"
            />
            <div
              className="labelCube redCube g5"
            />
            <div
              className="labelCube redCube g6"
            />
            <div
              className="labelCube redCube g7"
            />
            <div
              className="labelCube redCube g8"
            />
            <div
              className="labelCube redCube g9"
            />
            <div
              className="labelCube redCube g10"
            />
            <div
              className="labelCube redCube g11"
            />
            <div
              className="labelCube redCube g12"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterS"
          />
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube b1"
            />
            <div
              className="labelCube redCube b2"
            />
            <div
              className="labelCube redCube b3"
            />
            <div
              className="labelCube redCube b4"
            />
            <div
              className="labelCube redCube b5"
            />
            <div
              className="labelCube redCube b6"
            />
            <div
              className="labelCube redCube b7"
            />
            <div
              className="labelCube redCube b8"
            />
            <div
              className="labelCube redCube b9"
            />
            <div
              className="labelCube redCube b10"
            />
            <div
              className="labelCube redCube b11"
            />
            <div
              className="labelCube redCube b12"
            />
            <div
              className="labelCube redCube b13"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube o1"
            />
            <div
              className="labelCube redCube o2"
            />
            <div
              className="labelCube redCube o3"
            />
            <div
              className="labelCube redCube o4"
            />
            <div
              className="labelCube redCube o5"
            />
            <div
              className="labelCube redCube o6"
            />
            <div
              className="labelCube redCube o7"
            />
            <div
              className="labelCube redCube o8"
            />
            <div
              className="labelCube redCube o9"
            />
            <div
              className="labelCube redCube o10"
            />
            <div
              className="labelCube redCube o11"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube s1"
            />
            <div
              className="labelCube redCube s2"
            />
            <div
              className="labelCube redCube s3"
            />
            <div
              className="labelCube redCube s4"
            />
            <div
              className="labelCube redCube s5"
            />
            <div
              className="labelCube redCube s6"
            />
            <div
              className="labelCube redCube s7"
            />
            <div
              className="labelCube redCube s8"
            />
            <div
              className="labelCube redCube s9"
            />
            <div
              className="labelCube redCube s10"
            />
          </div>
           
        </div>
        <div>
           
          <div
            className="labelLetterM"
          >
            <div
              className="labelCube redCube s1"
            />
            <div
              className="labelCube redCube s2"
            />
            <div
              className="labelCube redCube s3"
            />
            <div
              className="labelCube redCube s4"
            />
            <div
              className="labelCube redCube s5"
            />
            <div
              className="labelCube redCube s6"
            />
            <div
              className="labelCube redCube s7"
            />
            <div
              className="labelCube redCube s8"
            />
            <div
              className="labelCube redCube s9"
            />
            <div
              className="labelCube redCube s10"
            />
          </div>
           
        </div>
      </div>
    </div>
    <div
      className="button-container button_start"
    >
      <div
        className="button-text shimmer_start"
      >
        start
      </div>
    </div>
  </div>
</div>
`;
